### PR TITLE
added a function that return online cpu number {map}

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

added a function that return on -line cpu number   
based on code :
https://github.com/autotest/autotest/blob/master/client/base_utils.py

### CODE REVIEW

File: avocado/utils/cpu.py [hunk 1]
```diff
@@ -114,3 +114,13 @@ def get_cpu_arch():
         return 'x86_64'
     else:
         return 'i386'
+
+def cpu_online_map():
+    """
+    Check out the available cpu online map
+    """
+    cpus = []
+    for line in open('/proc/cpuinfo', 'r').readlines():
```
Comment (ldoktor) [inline avocado/utils/cpu.py]: The use of `readlines()` is counterproductive as it creates list of all lines, while only `for line in open(...)` iterates the lines.

File: avocado/utils/cpu.py [hunk 2]
```diff
@@ -114,3 +114,13 @@ def get_cpu_arch():
         return 'x86_64'
     else:
         return 'i386'
+
+def cpu_online_map():
+    """
+    Check out the available cpu online map
```
Comment (ldoktor) [inline avocado/utils/cpu.py]: how about `Reports the online cpus as list`

Comment (PraveenPenguin) [inline avocado/utils/cpu.py]: @ldoktor  thanks looking into this . sure this make sense to Reports cpu as list will do .

File: avocado/utils/cpu.py (hunk 1 above)
Comment (PraveenPenguin) [inline avocado/utils/cpu.py]: sure thanks , will do changes .

**Comment by PraveenPenguin** [comment]
@ldoktor Raised a PR #1370  please have a look 
Regards
Praveen

**Pull request closed without merging**